qcut/docs/migration-tracker.md

```
@@ -0,0 +1,77 @@
+# Migration Tracking Spreadsheet
+
+## Next.js Dependencies Found
+
+### API Routes
+| File Path | Next.js Feature | Migration Target | Priority | Status |
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines

(MD058, blanks-around-tables)